state/allcollections.go

```diff
@@ -190,6 +190,12 @@ func allCollections() collectionSchema {
 			global: true,
 		},
 
+		// This collection holds information cached by autocert certificate
+		// acquisition.
+		autocertCacheC: {
+			global: true,
```

Comment:
done (BTW, tangential remark - the "modelUserLastConnectionC" collection below doesn't seem
to belong in this section as it's not global).